pypsa/examples.py

```diff
@@ -23,13 +24,36 @@
 logger = logging.getLogger(__name__)
 
 
+def _decrement_version(version):
+    x, y, z = map(int, version.split("."))
+    if z > 0:
+        z -= 1
+    elif y > 0:
+        y -= 1
+        z = 25  # TODO: This is a hack right now
+    elif x > 0:
+        x -= 1
+        y = z = 25  # TODO: This is a hack right now
+    return f"{x}.{y}.{z}"
+
+
 def _repo_url(master=False):
     url = "https://github.com/PyPSA/PyPSA/raw/"
     if master:
         return f"{url}master/"
-    from pypsa import __version__  # avoid cyclic imports
-
-    return f"{url}v{__version__}/"
+    from pypsa import release_version  # avoid cyclic imports
```

Comment:
Makes sense to have release version defined. The decrementing feels a bit hacky but should be okay :)

Reply:
Yeah I would change the decrement at some point. Either just going for the examples in the master branch, or going for examples for each version. But then they need to be updated in every version which is not happening right now.